cmd/buildah/commit.go

```diff
@@ -158,6 +158,10 @@ func commitCmd(c *cobra.Command, args []string, iopts commitInputOptions) error
 		}
 	}
 
+	// Add builder information.
+	builder.SetLabel("org.opencontainers.image.builder.name", "buildah")
+	builder.SetLabel("org.opencontainers.image.builder.version", buildah.Version)
```

Comment:
Based on @vbatts comment https://github.com/containers/buildah/issues/1864#issuecomment-547390122, I think that we should not use the `org.opencontainers.image` prefix to prevent users from assuming that's a supported annotation of the OCI image spec.

How about a single `io.buildah.version` label? It's presence shows both, that is was built with buildah _and_ with which version? @rhatdan @caiges @TomSweeneyRedHat @nalind WDYT?

Reply:
I concur on not using the org.opencontainers.image.  Where's the "io" from in your suggestion @vrothberg ?  My initial thought was "containers.image.*"

Reply:
I think that is a good idea.

Reply:
> I concur on not using the org.opencontainers.image. Where's the "io" from in your suggestion @vrothberg ? My initial thought was "containers.image.*"

The "io" comes from reversing the Buildah domain `buildah.io` -> `io.buildah` which might help consumers of an image find out what this label means. Maybe, just `io.buildah` as the key and have `vX.Y.Z` as the value?

Reply:
so it could be com.github.containers.buildah.version/commit too

Reply:
I like io.podman, nice and simple and easy to find.

Reply:
[The spec](https://github.com/opencontainers/image-spec/blob/master/annotations.md)'s rules suggest the reverse-the-order-of-dns-labels method that's used for Java packages, D-Bus services, and others.  This project doesn't "own" the `com.github` namespace prefix, and mustn't use the `org.opencontainers` prefix.

Reply:
I went with `io.buildah` for now since the podman/buildah relationship might be a little confusing if we were to use `io.podman`. I might be missing some context there though.